### ISSUE TITLE
APT whitelist request for musl

### ISSUE THREAD

**Issue by Downchuck** [travis-ci/travis-ci#3868]
It's a useful compiler. It would be nice to have a more recent version, due to the getaddrinfo bug in C-land.

**Comment by Downchuck** [comment]
Is any additional info/action required on my part?

**Comment by BanzaiMan** [comment]
Ubuntu repository doesn't have it. http://packages.ubuntu.com/precise/musl Where do you expect to find this package?

**Comment by BanzaiMan** [comment]
Did not run automated check because 'musl' needs trusty

**Comment by BanzaiMan** [comment]
Did not run automated check because 'musl' needs trusty

**Comment by Downchuck** [comment]
Blocked on trusty https://github.com/travis-ci/travis-ci/issues/2046

**Comment by BanzaiMan** [comment]
This APT source request has been moved to https://github.com/travis-ci/apt-package-whitelist/issues/369.

Moving forward, please create new requests at https://github.com/travis-ci/apt-package-whitelist/issues/new?title=APT+whitelist+request+for+___PACKAGE___

Thank you!
